gradle/check-licenses.gradle

```diff
@@ -113,19 +120,23 @@ downloadLicenses {
           license('Mozilla Public License Version 1.1', 'http://www.mozilla.org/media/MPL/1.1/index.0c5913925d40.txt')
       ],
       (mpl2_0): [
+          'Mozilla Public License, Version 2.0',
           'MPL 2.0',
           'Mozilla Public License Version 2.0',
           license('Mozilla Public License 2.0', 'http://www.mozilla.org/media/MPL/2.0/index.815ca599c9df.txt')
       ],
       (cddl): [
               'CDDL',
+              'CDDL/GPLv2+CE',
+              'Common Development and Distribution License (CDDL) v1.0',
               'Common Development and Distribution License 1.0',
               'CDDL + GPLv2 with classpath exception',
+              'GPLv2+CE',
```

Comment:
This isn't the CDDL.  it's GPLv2 + Classpath Exception.  It's allowed, but shouldn't be listed in the CDDL list (whereas CDDL/GPLv2+CE is ok here because it means either CDDL or GPLv2+CE so we'd pick CDDL).

Reply:
excluded the library that uses it...